components/layout_2020/dom.rs

```diff
@@ -43,6 +44,7 @@ pub(super) enum LayoutBox {
     BlockLevel(ArcRefCell<BlockLevelBox>),
     InlineLevel(ArcRefCell<InlineItem>),
     FlexLevel(ArcRefCell<FlexLevelBox>),
+    TableBox(TableBox),
```

Comment:
I'm thinking that this can be confused to refer to the box generated by the table itself (in fact "table box" is the term used in https://www.w3.org/TR/2011/REC-CSS2-20110607/tables.html#model).

May be better to use `InternalTableBox` (defined in https://drafts.csswg.org/css2/#internal-table-box and https://drafts.csswg.org/css-display-3/#internal-table-box, technically it doesn't include captions but probably fine) or `TableLevel` (I think no spec uses this term but for consistency with the other *-level it should refer to a box that participates in a table formatting context).

Both for the `LayoutBox` entry and for the nested enum.